src/cmd/getM365/exchange/getItem.go

```diff
@@ -24,52 +24,36 @@ import (
 	"github.com/alcionai/corso/src/internal/data"
 	"github.com/alcionai/corso/src/pkg/account"
 	"github.com/alcionai/corso/src/pkg/credentials"
-	"github.com/alcionai/corso/src/pkg/logger"
 	"github.com/alcionai/corso/src/pkg/path"
 )
 
-var getCmd = &cobra.Command{
-	Use:   "get",
-	Short: "Get a M365ID item JSON",
-	RunE:  handleGetCommand,
-}
-
 // Required inputs from user for command execution
 var (
-	tenant, user, m365ID, category string
+	user, tenant, m365ID, category string
 )
 
-// main function will produce the JSON String for a given m365 object of a
-// user. Displayed Objects can be used as inputs for Mockable data
-// Supports:
-// - exchange (contacts, email, and events)
-// Input: go run ./getItem.go     --user <user>
-//
-//	--m365ID <m365ID> --category <oneof: contacts, email, events>
-func main() {
-	ctx, _ := logger.SeedLevel(context.Background(), logger.Development)
-	ctx = SetRootCmd(ctx, getCmd)
-
-	defer logger.Flush(ctx)
-
-	fs := getCmd.PersistentFlags()
-	fs.StringVar(&user, "user", "", "m365 user id of M365 user")
-	fs.StringVar(&tenant, "tenant", "",
-		"m365 Tenant: m365 identifier for the tenant, not required if active in OS Environment")
+func AddCommands(parent *cobra.Command, userFlag, tenantFlag string) {
+	user = userFlag
+	tenant = tenantFlag
```

Comment:
Not sure if you're trying to link references, or copy values, but either way it won't work.  Per references, the flag pointers wont update the exchange package vars.  Per values, flag values are not populated until command execution.

If you want to distribute a common set of flag vars, they'll need to go into a third package.  For example, see: src/cli/options/options.go.

Reply:
🤦 - I should really test this code before pushing.